File: Lang - C/guess-number.c
#include <stdio.h>
#include <time.h>
#include <math.h>

int main(void) {
    int cur_time = time(NULL) >> 2;
    int random_num = abs((cur_time * time(NULL) % 15) - 4);
    int user_guess;
    while (user_guess != random_num)
    {
        printf("Please enter a number from 0 to 9\n");
        scanf("%d", &user_guess);
        if (user_guess > 10 || user_guess < 0)
        {
            printf("That's too far out of scope\n");
        }
        else if (user_guess > random_num) {
            printf("That's a little too high\n");
        }
        else if (user_guess < random_num) {
            printf("That's a little too low\n");
        }
    }
    printf("That's correct! Well done!\n");
    return 0;
}

File: Lang - JS-TS/Lang - TypeScript/guess-number/guess-number.ts
const readline = require("readline")

// Create the input
var random_num = Math.floor(Math.random()*10)
var user_num = 10
console.log(random_num)

function main() {
    const reader = readline.createInterface( process.stdin, process.stdout )
    console.log("Please enter a number from 0 to 9")
    reader.question(">> ", user_input => {
            console.log(user_input)
            user_num = parseInt(user_input)
            reader.close()
            if (user_num == random_num) {
                console.log("That's correct! Well done!")
                return
            }
            else if (user_num > random_num) {
                console.log("That's a little too high")
            }
            else {
                console.log("That's a little too low")
            }
            main()
            return
        }
    )
}

main()